flame/star/star_model_tester.py

```
@@ -12,6 +11,8 @@ class StarModelTester:
     num_iterations: int = 0
     converged: bool = False
 
+    sim_nodes: dict[str, Union[StarModel, StarLocalDPModel]] = {}
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)